apps/demos/testing/widgets/datagrid/ToolbarCustomization.test.js

```diff
@@ -0,0 +1,39 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
+
+fixture('DataGrid.ToolbarCustomization')
+  .page('http://localhost:8080/')
+  .before(async (ctx) => {
+    ctx.initialWindowSize = [900, 600];
+  });
+
+runManualTest('DataGrid', 'ToolbarCustomization', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test.only('ToolbarCustomization', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+    const menuButtonSelector = $('.dx-toolbar').find('.dx-toolbar-menu-container').find('.dx-button')
+    const menuItemsSelector = $('.dx-overlay-wrapper').find('.dx-list-item')
+
+    await testScreenshot(t, takeScreenshot, 'toolbar_customization_wide.png');
+    
+    await t
+      .resizeWindow(480, 600)
+      .click(menuButtonSelector)
+      .expect(menuItemsSelector.count).eql(3);
+
+    await testScreenshot(t, takeScreenshot, 'toolbar_customization_medium.png');
+
+    await t
+      .click($('body'), { offsetX: 0, offsetY: 0 })
+      .resizeWindow(320, 600)
```

Comment:
(just for knowledge sharing, not applicable to current PR)
Don't forget to restore initial sizes of window after the test. Usually we use our `safeSiteTest` utility instead of test. It can't be used in demos testing for now unfortunately, and as I see in every demo test we set `ctx.initialWindowSize = [900, 600];` in the beginning, so no need to change anything here